gulpfile.js

```diff
@@ -55,6 +51,9 @@ gulp.task('bundle-debug', function() {
 
   var bundle = function() {
     return bundler
+      .transform(preprocessify(
+        { DEBUG: true }
```

Comment:
What about having `PERF` and `DEBUG` so in production we can have both disabled and remove performance marks as well as debug traces? If you like, open a new issue with the change.